pinecone/utils/version.py

```diff
@@ -3,6 +3,4 @@
 def get_version():
     return Path(__file__).parent.parent.joinpath("__version__").read_text().strip()
 
-
-def get_environment():
-    return Path(__file__).parent.parent.joinpath("__environment__").read_text().strip()
\ No newline at end of file
+__version__ = get_version()
```

Comment:
There's really no reason for anyone outside this file to use `get_version()` since it just leads to reading from disk multiple times unnecessarily.